polaris-react/src/styles/shared/_buttons.scss

```diff
@@ -28,7 +28,7 @@
   #{$se23} & {
     min-height: 28px;
     min-width: 28px;
-    padding: var(--p-space-1) var(--p-space-3);
+    padding: var(--p-space-1_5-experimental) var(--p-space-3);
```

Comment:
This makes buttons with icons with icons 32px in height vs the desired 28px. You might need to play with the line height or something instead of changing base button padding

Reply:
> Thanks for catching that I missed it since it visually looked ok in the `CalloutCard` examples.
> 
> What was the reason for the padding change @gwyneplaine?

@sarahill button vertical padding in figma is `6px` rather than `4px`, noticed this in the CalloutCard designs and double checked it in the Button designs 

happy to walk this back given @kyledurand's point about buttons \w icons 👍